docs/source/software/r.rst

```diff
@@ -2,19 +2,20 @@
 Working with R
 ***************
 `R <https://www.r-project.org/>`_ is available as a module (see :ref:`using_module` for more information) and
-it is also an interactive app on Open onDemand (see :ref:`using_ood` for more information). You can also use R with Anaconda. See :ref:`working_conda` and the `Anaconda documentation <https://docs.anaconda.com/anaconda/packages/r-language-pkg-docs/>`_ for more information
+Rstudio is also an interactive app on Open onDemand (see :ref:`using_ood` for more information). You can also use R with Anaconda. See :ref:`working_conda` and the `Anaconda documentation <https://docs.anaconda.com/anaconda/packages/r-language-pkg-docs/>`_ for more information
 
 If you work with R packages, using a Packrat environment can be helpful. Use the procedure below to create a Packrat environment on Discovery.
 
 Creating a Packrat Environment (for R)
 ======================================
 
-Packrat is an application that helps you manage packages for R. After you create a new directory for your R project, you can then use Packrat
-to store your package dependencies inside it. For more information about Packrat, see the website: https://rstudio.github.io/packrat/.
+Packrat is an application that helps you manage packages for R. Packrat allows for greater reproducibility because it saves the exact package version that you installed, it's portable from one machine to another (or node-to-node in HPC land), and it's isolated, allowing you to create separate project directories and avoid potential package conflicts. After you create a new directory for your R project, you can then use Packrat to store your package dependencies inside it. For more information about Packrat, see the website: https://rstudio.github.io/packrat/.
+
+**When using R in the command line:**
 
 1. Connect to Discovery.
 2. Type ``module load R/3.6.2``.
```

Comment:
We could probably raise the version of R to a newer version of R because some might just follow the steps verbatim.